### ISSUE TITLE
Using playwright-chromium

### ISSUE THREAD

**Issue by celeryclub** [qawolf/qawolf#574]
Hey team. Hope all is well!

I'm curious if you plan to support using `playwright-chromium` or another sub-package instead of just the full `playwright` package. I'm seeing this warning right now, so it looks like this type of usage isn't supported.

```sh
npm WARN qawolf@0.13.1 requires a peer of playwright@^0.13.0 but none is installed. You must install peer dependencies yourself.
```

This is something that would be useful for me, and I'm curious if there are any specific limitations that are blocking this. Thanks!

**Comment by jperl** [comment]
It's a warning but it should still work. I will remove the warning in the next release though.

**Comment by jperl** [comment]
Actually launch will not work. I will make sure we fix that though in this ticket, please use `playwright` in the meantime.

**Comment by celeryclub** [comment]
Yes that is what I was seeing. Thank you for addressing this! 🙌

**Comment by jperl** [comment]
@celeryclub This is fixed in `qawolf@0.13.2`.

**Comment by celeryclub** [comment]
Can confirm that this is working now. Thanks @flaurida @jperl!
